### DISCUSSION TITLE
Set default email for web based git operations

### DISCUSSION THREAD

**Discussion by mahdyar** [community/community#5985] [Pull Requests]
If you suggest a change in a PR and the author decides to commit the suggestion from GitHub UI, It'll use my primary email, which I don't use for my git commits. If there were an option to choose default web based git operations email it'd be much better.

**Comment by huehnerlady** [comment]
I agree. I find it being 2 different things. your primary address is for resetting passwords and for all out-of-organizations notifications.

The web-based operations are something different.
I would like to add that setting an email per organization is important as well as you might have different emails for different organizations (which you have in the notification section already)

**Comment by belmind** [comment]
Great suggestion! I had to email Github support in order for them to update it :)

**Reply by huehnerlady** [reply]
@BelminD how do you mean? Can you actually get that set up by organization if you contact support?

**Reply by belmind** [reply]
> @BelminD how do you mean? Can you actually get that set up by organization if you contact support?
> 
> 

Correct :)

**Reply by huehnerlady** [reply]
ah cool thanks, will contact support straight away then :)
